src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/VODCreation/PublishVOD/index.tsx

```diff
@@ -115,12 +116,9 @@ export const PublishVOD = () => {
                   iconColor={normalizeColor('blue-active', theme)}
                   height="20px"
                 />
-                <Paragraph
-                  margin={{ left: 'small', vertical: 'auto' }}
-                  color="blue-active"
-                >
+                <Text className="ml-t">
```

Comment:
Type p ?

Reply:
In this case it is a text near a icon, so we want to have an inline element, span should be the natural html choice.